sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/PostgresTestOptions.java

```diff
@@ -0,0 +1,40 @@
+package org.apache.beam.sdk.io.jdbc;
+
+import org.apache.beam.sdk.options.Default;
+import org.apache.beam.sdk.options.Description;
+import org.apache.beam.sdk.testing.TestPipelineOptions;
+
+/**
+ * These options can be used by a test connecting to a postgres database to configure their connection.
+ */
+public interface PostgresTestOptions extends TestPipelineOptions {
```

Comment:
Can these be in the test package? Even though they are data-source based, I don't believe we should be exposing these to any users of `JdbcIO`